[1,"=> shiny::runApp()\n\n"]
[2,"Loading required package: shiny\n"]
[2,"\nAttaching package: ‘shinydashboard’\n\nThe following object is masked from ‘package:graphics’:\n\n    box\n\n"]
[2,"\nAttaching package: ‘dplyr’\n\nThe following objects are masked from ‘package:stats’:\n\n    filter, lag\n\nThe following objects are masked from ‘package:base’:\n\n    intersect, setdiff, setequal, union\n\n"]
[2,"\nAttaching package: ‘DT’\n\nThe following objects are masked from ‘package:shiny’:\n\n    dataTableOutput, renderDataTable\n\n\nInitialize Python Version 2.7.16 (default, Jun  5 2020, 22:59:21) \n[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.29.20) (-macos10.15-objc-\n\n"]
[2,"Using poppler version 0.73.0\n"]
[1,"R is already connected to Python!\n"]
[2,"\nListening on http://127.0.0.1:3790\n"]
[1,"Traceback (most recent call last):\n\n  File \"helper.py\", line 10, in get_best_match\n\n    \nimport pandas as pd\n\nImportError\n: \nNo module named pandas\n"]
[2,"Warning: Error in pyCall: error occured while calling 'get_best_match'\n"]
[2,"  103: \u001B[37mpyCall\u001B[39m\n  102: \u001B[34m\u001B[1mexprFunc [/Users/work/js1/RShiny/app.R#64]\u001B[22m\u001B[39m\n  101: \u001B[37mwidgetFunc\u001B[39m\n  100: \u001B[37mfunc\u001B[39m\n   87: \u001B[37morigRenderFunc\u001B[39m\n   86: \u001B[37mrenderFunc\u001B[39m\n   82: \u001B[37morigRenderFunc\u001B[39m\n   81: \u001B[37moutput$table\u001B[39m\n    1: \u001B[37mshiny::runApp\u001B[39m\n"]
